app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/AddItem.java

```diff
@@ -0,0 +1,66 @@
+package com.example.avjindersinghsekhon.minimaltodo.testApp.tests;
+
+import android.widget.EditText;
+import android.widget.ImageButton;
+
+import com.example.avjindersinghsekhon.minimaltodo.R;
+
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withParent;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.startsWith;
+
+public class AddItem {
+
+    public static boolean isXButton() {
+        return Helpers.checkIfUIObjectIsVisible(allOf(is(instanceOf(ImageButton.class)), withParent(withId(R.id.toolbar))));
+    }
+
+    public static void clickXButton() {
+        Helpers.clickEl(allOf(is(instanceOf(ImageButton.class)), withParent(withId(R.id.toolbar))));
+    }
+
+    public static boolean isFloatingActionButton() {
+        return Helpers.checkIfUIObjectIsVisible(withId(R.id.toDoCustomTextInput));
+    }
+
+    public static boolean isEditText() {
+        return Helpers.checkIfUIObjectIsVisible(allOf(is(instanceOf(EditText.class)), withParent(withId(R.id.toDoCustomTextInput))));
+    }
+
+    public static void clickSwitchOn() {
```

Comment:
If I use this method to click the switch twice, it will be always "On"?

Reply:
No, when I click the button is turning switch on or off with every click.